src/allencell_ml_segmenter/main/viewer.py

```diff
@@ -105,3 +116,47 @@ def _get_layer_by_name(self, name: str) -> Optional[Layer]:
             if l.name == name:
                 return l
         return None
+
+    def get_seg_layers(self, layer_list: list[Layer]) -> list[Layer]:
```

Comment:
Appears that `layer_list` parameter is unused in the implementation -> prob fine to get rid of it in the `i_viewer` and implementations